### DISCUSSION TITLE
Ayuda (Diseño de Abogabot)

### DISCUSSION THREAD

**Discussion by NestorMyrie** [Launch-X-Latam/MisionFrontEnd#207] [Q&A]
Buenas tardes Devs, ocupo sus opiniones, necesito saber si entendí bien la idea del abogabot, este es el diseño en el que estoy trabajando, voy por la Fase 2 de 3 y antes de seguir avanzando me gustaria saber si si va asi , gracias!!

[DISEÑO EN FIGMA ](https://www.figma.com/file/J8JKpoBfMEK0eCBMvqSrgu/Flipbook-Animator-GIF-(Community)?node-id=39%3A986)

**Comment by EsthebanDiego** [comment]
También voy empezando, pero yo lo veo muy bien 👍
